src/rastervision/ml_tasks/object_detection.py

```diff
@@ -65,8 +65,11 @@ def get_train_windows(self, project, options):
         # any labels. It may take many attempts to generate a single
         # negative window, and could get into an infinite loop in some cases,
         # so we cap the number of attempts.
-        nb_neg_windows = \
-            int(options.object_detection_options.neg_ratio * len(pos_windows))
+        if nb_pos_windows:
```

Comment:
This needs to be preceded by `nb_pos_windows = len(pos_windows)`

Reply:
Yes... you're correct...